src/config/constants.ts

```diff
@@ -25,3 +25,8 @@ export const SAFE_TOKEN_ADDRESSES: { [chainId: string]: string } = {
   '1': '0x5aFE3855358E112B5647B952709E6165e1c1eEEe',
   '4': '0xCFf1b0FdE85C102552D1D96084AF148f478F964A',
 }
+
+// Safe Apps
+export const SAFE_POLLING_INTERVAL = process.env.NODE_ENV === 'test' ? 4500 : 15000
```

Comment:
If the comment is removed, it will be hard to guess what this variable is used for... Maybe we can name everything related to safe apps with the `SAFE_APPS_` prefix

Reply:
Also there's already a POLLING_INTERVAL on line 7, maybe they should be the same?

Reply:
done